src/test/java/net/minecraftforge/debug/block/CustomRespawnTest.java

```diff
@@ -87,9 +87,9 @@ public InteractionResult use(BlockState state, Level world, BlockPos pos, Player
         }
 
         @Override
-        public Optional<Vec3> getRespawnPosition(BlockState state, EntityType<?> type, LevelReader world, BlockPos pos, float orientation, @Nullable LivingEntity entity)
+        public Optional<Vec3> getRespawnPosition(BlockState state, EntityType<?> type, LevelReader levelReader, BlockPos pos, float orientation, @Nullable LivingEntity entity)
```

Comment:
Other uses of `LevelReader` seem to use `level` instead of `levelReader`.

Reply:
This was named as such since the parent (`IForgeBlock#getRespawnPosition`) uses the name `level` for a local variable of type `Level`.